app/src/main/java/edu/uc/hickmadc/customnotifications/MainActivity.kt

```diff
@@ -14,8 +14,8 @@ import edu.uc.hickmadc.customnotifications.ui.main.MyListAdapter
 class MainActivity : AppCompatActivity() {
 
     //temp data
-    val language = arrayOf<String>("Brush Teeth","Eat Breakfast","Do schoolwork")
-    val description = arrayOf<String>(
+    private val language = arrayOf<String>("Brush Teeth","Eat Breakfast","Do schoolwork")
```

Comment:
changed to private val

Reply:
This duplicates a previous code review.  When that occurs, first in wins.